src/loop.jl

```diff
@@ -75,6 +80,9 @@ function interval_loop(factory_like, model_kwargs::Dict,
             if storage.enabled
                 model_kwargs["storage_e0"] = storage_e0
             end
+            if demand_flexibility.enabled
+                model_kwargs["init_shifted_demand"] = init_shifted_demand
+            end
```

Comment:
Here we are assuming implicitly if demand flexibility is enabled, at least one of interval balance and rolling balance is enabled?

Reply:
Done. I had actually kind of accounted for this on line 267 of `loop.jl`. I  also added an `else` statement there to catch the times that both interval balance and rolling balance are not implemented.